packages/react-core/src/demos/Button.md

```diff
@@ -0,0 +1,75 @@
+---
+id: Button
+section: components
+---
+
+import CheckCircleIcon from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
+
+## Demos
+
+### Progress button
+
+```ts
+import React from 'react';
+import { Form, FormGroup, ActionGroup, InputGroup, TextInput, Button } from '@patternfly/react-core';
+import CheckCircleIcon from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
+
+const ProgressButton = () => {
+  const [loginState, setLoginState] = React.useState<'notLoggedIn' | 'loading' | 'loggedIn'>('notLoggedIn');
+  const actionButton = {
+    notLoggedIn: (
+      <Button
+        variant="primary"
+        onClick={() => {
+          setLoginState('loading');
+          setTimeout(() => {
+            setLoginState('loggedIn');
+          }, 3000);
+        }}
+      >
+        Link account and log in
+      </Button>
+    ),
+    loading: (
+      <Button variant="primary" aria-label="Loading..." isLoading>
```

Comment:
From the existing progress example, looks like we add an `aria-label` to the spinner via `spinnerAriaValueText` instead of adding it to the button element.

```suggestion
      <Button variant="primary" spinnerAriaValueText="Loading..." isLoading>
```

Reply:
Thanks for the info! changed it